app/controllers/static_page_controller.rb

```diff
@@ -0,0 +1,5 @@
+class StaticPageController < ApplicationController
+  def home
+    @products = Product.paginate(:page => params[:page], :per_page => 6).order("created_at desc")
```

Comment:
```suggestion
    @products = Product.paginate(page: params[:page], per_page: 6).order("created_at desc")
```
order viết scope em nhé